test/test_prototype_transforms_consistency.py

```diff
@@ -655,6 +655,15 @@ def test_call_consistency(config, args_kwargs):
     )
 
 
+@pytest.mark.parametrize(
+    "config",
+    [config for config in CONSISTENCY_CONFIGS if hasattr(config.legacy_cls, "get_params")],
+    ids=lambda config: config.legacy_cls.__name__,
+)
+def test_get_params_consistency(config):
+    assert config.prototype_cls.get_params is config.legacy_cls.get_params
```

Comment:
To be on the safe side, could we also check that a) we can actually call `get_params()` and that they're jit-scriptable?
Perhaps no need to test them all as it might be tricky considering they don't all have the same signature, but a few representative cases could be enough?